GroundControl.gd

```diff
@@ -0,0 +1,67 @@
+extends Node2D
+
+var o2_button = 0
+var accuracy_button = 0
+var attack_button = 0
+var speed_button = 0
+var defense_button = 0
+var dodge_button = 0
+
+func _ready():
+	$CanvasLayer/Accuracy.visible = false
+	$CanvasLayer/Speed.visible = false
+	$CanvasLayer/Defense.visible = false
+	$CanvasLayer/Attack.visible = false
+	$CanvasLayer/Dodge.visible = false
+	$CanvasLayer/o2_1.visible = false
+	$CanvasLayer/o2_2.visible = false
+	$CanvasLayer/as_1.visible = false
+	$CanvasLayer/as_2.visible = false
```

Comment:
As I mentioned above, let's stick with full word descriptors since that's what you did that in most places.